official/NormeDiProgetto/res/sections/organizativeProcess.tex

```diff
@@ -25,6 +25,15 @@ \subsubsection{Attività di pianificazione}
 
 \paragraph*{Assegnazione dei \glossaryItem{task}} Il responsabile si occuperà di assegnare i \glossaryItem{task} in base alla disponibilità dei membri del gruppo e alla rotazione dei ruoli.
 
+\paragraph*{Assegnazione verifica delle pull request} Le pull request create dall'assegnatario di un task per la verifica sono assegnate in automatico. L'assegnatario di un task, nel momento di creazione di una Pull Ruquest, dovrà lasciare vuoto il campo \textit{Assignee}. Il procedimento d'assegnazione dovrà valutare il verificatore in base ai seguenti criteri:
+\begin{itemize}
+\item Il verificatore della Pull Request deve essere diverso dall'assegnatario del task.
+\item Il verificatore è un membro del gruppo \textit{BugBusters} con il numero minore di Pull Request assegnate.
+\item In caso di riscontri multipli sarà selezionato il primo membro identificato come idoneo.
+\end{itemize}
+
+Il fine \`e di diminuire il tempo di verifica rimuovendo il tempo richiesto dal responsabile per l'assegnazione della Pull Request.
+
```

Comment:
Secondo me sarebbe utile dire con cosa vengono assegnate le PR in automatico, cioè quale parte della nostra infrastruttura se ne fa carico